db_migrator--1.1.0.sql

```diff
@@ -38,11 +37,7 @@ $$DECLARE
    v_default          boolean;
 BEGIN
    /* set "search_path" to the PostgreSQL stage, and extension schema */
-   SELECT extnamespace::regnamespace::text INTO extschema
-   FROM pg_extension
-   WHERE extname = 'db_migrator';
-
-   EXECUTE format('SET LOCAL search_path = %I, %s', pgstage_schema, extschema);
+   EXECUTE format('SET LOCAL search_path = %I, @extschema@', pgstage_schema);
```

Comment:
Wouldn't it be even simpler to change the function definitions to

    CREATE FUNCTION ....
       LANGUAGE plpgsql
       SET search_path = @extschema@ AS
    $$...$$;

Reply:
Unfortunaly, it does not work:
```
+ERROR:  function adjust_to_bigint(numeric) does not exist
+LINE 5:           adjust_to_bigint(min_value),
```
`SET LOCAL search_path` seems to override function' search_path

Reply:
You are right. I didn't think enough.